src/python/pants/core/util_rules/environments.py

```diff
@@ -201,6 +201,34 @@ class DockerImageField(StringField):
         """
     )
 
+class DockerBindMountsField(StringSequenceField):
+    alias = "bind_mounts"
+    help = help_text(
+        """
+        Read-only bind mounts to use when starting the container.
+
+        The values should be in the form "<path on host>:<path on container>"
+        """
+    )
+
+    @classmethod
+    def compute_value(
+        cls, raw_value: Optional[Iterable[str]], address: Address
+    ) -> Optional[Tuple[str, ...]]:
+        values = super().compute_value(raw_value, address)
+        if values and (bad_values := [val for val in values if len(val.split(":")) != 2]):
```

Comment:
It looks like these are basically just passed through to docker directly; could we instead rely on its validation rather than Pants imposing its own rules?

Reply:
We could, yeah.

The more I think about it, the more I like that idea.